src/protocol.h

```diff
@@ -327,12 +327,16 @@ std::vector<std::string> serviceFlagsToStr(uint64_t flags);
  * guaranteed to not change dependent on state - ie they are suitable for
  * use when describing peers which we know to be desirable, but for which
  * we do not have a confirmed set of service flags.
- *
- * If the NODE_NONE return value is changed, contrib/seeds/makeseeds.py
```

Comment:
Was it just outdated already? I'm confused by what NODE_NONE meant here before this change.

Reply:
> Was it just outdated already? I'm confused by what NODE_NONE meant here before this change.

It was talking about the input `ServiceFlags`. Not the `GetDesirableServiceFlags()` output.

Basically, we expect to have all the hardcoded seeds supporting `NODE_NETWORK` and `NODE_WITNESS`.